MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,133 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
 {% endhighlight %}
 {% endtabs %}
 
+## Customize the form fields programmatically
+
+Customizing form fields is crucial for controlling their appearance and behavior. By modifying properties like BackgroundColor, BorderColor, BorderWidth, and ForegroundColor, you can tailor how each form field (such as TextFormField, CheckboxFormField, RadioButtonFormField, etc.) looks within the PDF viewer. Like other form fields properties, you can undo and redo the hiding or showing actions. Additionally, when an form field is locked, it cannot be hidden. Form fields that are hidden will remain hidden during import, export, printing and saving.
```

Comment:
Why are you explaining about hide and show feature here?
<img width="509" alt="image" src="https://github.com/user-attachments/assets/b3664cd9-3769-4131-b0cc-beb156731b84" />